### DISCUSSION TITLE
nvim-cmp not updating with packer

### DISCUSSION THREAD

**Discussion by RayZ0rr** [hrsh7th/nvim-cmp#742] [General]
Just tried `PackerSync` and nvim-cmp failed

**Comment by RayZ0rr** [comment]
Removing `nvim-cmp `folder and installing again fixed it.
`rm -rf ~/.local/share/nvim/site/pack/packer/start/nvim-cmp`
and
`:PackerSync`

**Reply by smjonas** [reply]
Fyi, this was probably caused by a force-push to the main branch. An alternative way to solve this - should this occur again - is to navigate to the root directory of the plugin and run `git pull -f`.

**Reply by RayZ0rr** [reply]
Oh. Thanks for letting me know.
